crates/swift-bridge-cli/src/clap_exec.rs

```diff
@@ -0,0 +1,41 @@
+use std::collections::HashMap;
+use std::path::{Path};
+use clap::ArgMatches;
+use swift_bridge_build::{ApplePlatform, create_package, CreatePackageConfig};
+
+/// Executes the correct function depending on the cli input
+pub fn handle_matches(matches: ArgMatches) {
+    match matches.subcommand_name() {
+        Some("create-package") => handle_create_package(matches.subcommand_matches("create-package").unwrap()),
+        _ => unreachable!("No subcommand or unknown subcommand given") // Shouldn't happen
+    }
+}
+
+/// Executes the `create-package` command
+fn handle_create_package(matches: &ArgMatches) {
+    let bridges_dir = matches.value_of("bridges-dir").unwrap(); // required
+    let out_dir = matches.value_of("out-dir").map(|p| Path::new(p)).unwrap(); // required
+    let name = matches.value_of("name").unwrap_or(out_dir.file_name().expect("Couldn't derive Swift Package name from output directory").to_str().unwrap());
```

Comment:
Using the name of the directory as the package name feels a bit unexpected.

Can we start by just making name a required field and if in the future it ends up feeling redundant we can consider a shortcut like this?